hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmLCExpiration.java

```diff
@@ -38,6 +38,7 @@
 public final class OmLCExpiration implements OmLCAction {
   private final Integer days;
   private final String date;
+  private final ZonedDateTime createDate;
```

Comment:
@xichen01 , thanks for working on this.  How about we persist this createDate into OmLCExpiration proto, so we can assume createDate is always there, and use it to verify the expiry date set by user, so we don't need to decide whether it should call buildAndValid or build, leave no room for call the function incorrectly?

Reply:
I have directly use the `OmLifecycleConfiguration#creationTime`, this has been a persist field, ad we do not use it previous. 
A Lifecycle may contains multiple Rule, both these Rule should have a same creation time, so put the creationTime on the OmLifecycleConfiguration is appropriate.